### PR TITLE
Add user menu and helpers.

### LINKED ISSUE #276: Add user information to indicate currently logged in user
Add the name of the logged-in user in master layout either on top of tab bar. i.e. the navigation menu. 
Refer to
 wireframe/V1/OrganizationalView1.1.png)

As a user I want to be recognised, when I'm logged in so that I can easily see who I'm logged in as.

# Definition of done
- [ ] Name of logged-in user (if available) is displayed in navigation menu. If the user has not given a name, show the email.

### PR DESCRIPTION
**Pull request by jykae**
Closes #276 

# Proposed changes
- add user menu in AdminLTE style

@Digipalvelutehdas/dipor-team please review.

### CODE REVIEW

File: core/client/layouts/main/navigation/navigation.js [hunk 1]
```diff
@@ -4,3 +4,20 @@ Template.mainLayoutNavigation.events({
     AccountsTemplates.logout();
   }
 });
+
+Template.mainLayoutNavigation.helpers({
+  "currentUserName": function () {
```
Comment (elnzv) [inline core/client/layouts/main/navigation/navigation.js]: Add comments

File: core/client/layouts/main/navigation/navigation.js [hunk 2]
```diff
@@ -4,3 +4,20 @@ Template.mainLayoutNavigation.events({
...
+    if( Meteor.user().profile && Meteor.user().profile.name !== undefined ) {
+      return Meteor.user().profile.name;
+    } else {
+      return false;
+    }
+  },
+  "currentUserEmail": function () {
+    if( Meteor.user().emails[0] ) {
```
Comment (elnzv) [inline core/client/layouts/main/navigation/navigation.js]: And comments here

**Comment by elnzv** [comment]
Please comments your js code. I did a small change to links position. Otherwise looks good.

**Comment by jykae** [comment]
@elnzv Thanks for CSS fixes, I add comments to my code.

**Comment by jykae** [comment]
@elnzv Added comments, please review and merge if ok.

**Comment by brylie** [comment]
The menu seems to be invisible on mobile display. E.g. when the navigation menu collapses to a 'hamburger' the user menu is hard to see.

**Comment by jykae** [comment]
@brylie thanks for finding that :)

**Comment by jykae** [comment]
Maybe I could remove that dropdown, we have only that useless Home link there, what you think @elnzv @brylie ?

nayttokuva 2016-05-06 kello 14 40 10

**Comment by jykae** [comment]
@Digipalvelutehdas/dipor-team Review and merge if ok.

**Pull request merged**